cloud/logger-cloud/src/Logger.ts

```diff
@@ -0,0 +1,11 @@
+
+
+export interface Logger {
+    //log(message:string, object:any): void;
+    debug(message:string, object:any): void;
```

Comment:
I think we can remove the `: void` return values here, might make it more convenient to implement it with a weird library that for some reason returns something.